native/packages/react-native-bpk-component-badge/readme.md

```diff
@@ -0,0 +1,54 @@
+# react-native-bpk-component-badge
+
+> Backpack React Native badge component.
+
+## Installation
+
+```sh
+npm install react-native-bpk-component-badge --save-dev
+```
+
+## Usage
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet } from 'react-native';
+import BpkBadge from 'react-native-bpk-component-badge';
+import { spacingBase } from 'bpk-tokens/tokens/base.react.native';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    padding: spacingBase,
+  }
+});
+
+export default class App extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <BpkBadge message="Badge" type="success" />
+        <BpkBadge message="Badge" type="warning" />
+        <BpkBadge message="Badge" type="destructive" />
+        <BpkBadge message="Badge" type="inverse" />
+        <BpkBadge message="Badge" type="light" />
+        <BpkBadge message="Badge" type="outline" />
+        <BpkBadge message="Badge" type="success" />
+        <BpkBadge message="Badge" docked="left" type="warning" />
+        <BpkBadge message="Badge" docked="right" type="destructive" />
+        <BpkBadge message="Badge" type="inverse" />
+        <BpkBadge message="Badge" docked="left" type="light" />
+        <BpkBadge message="Badge" docked="right" type="outline" />
```

Comment:
I'm wondering if it'll be better to make these examples use `BADGE_TYPES` - what do you think?

Reply:
to iterate through them? Nah this code should be as straightforward as possible.

Reply:
Not to iterate, I meant instead of `type="light"`, `type={BADGE_TYPES.light}`.

However I've just looked at our readmes and we don't do that in others anyway, so it doesn't matter so much.

Reply:
Ah in that case I agree :)